genapkovl-lima.sh

```diff
@@ -240,6 +240,18 @@ if [ "${LIMA_INSTALL_SSHFS}" == "true" ]; then
     echo "sshfs" >> "$tmp"/etc/apk/world
 fi
 
+if [ "${LIMA_INSTALL_CRI_DOCKERD}" == "true" ]; then
+    mkdir -p "${tmp}/cri-dockerd"
+    tar xz -C "${tmp}/cri-dockerd" -f /home/build/cri-dockerd.tar.gz
+    chmod u+s "${tmp}/cri-dockerd/cri-dockerd"
```

Comment:
I realized that we don't need this, so please remove.

I understand that you just copied from the nerdctl section above. We mark those setuid so they can be invoked without `sudo`, but then RD doesn't seem to take advantage of that (yet?).

But `cri-dockerd` should always be started by `/etc/init.d/cri-dockerd` so is already root.

Reply:
That makes sense. Thanks